windows_containers/wmco_rn/windows-containers-release-notes-7-x-past.adoc

```diff
@@ -117,11 +99,12 @@ The following known limitations have been announced after the previous WMCO rele
 
 * {productwinc} does not support adding Windows nodes to a cluster through a trunk port. The only supported networking configuration for adding Windows nodes is through an access port that carries traffic for the VLAN.
 
-* WMCO 7.0.0 does not support running in a namespace other than `openshift-windows-machine-operator`. If you are using a custom namespace, it is recommended that you not upgrade to WMCO 7.0.0. Instead, you should upgrade to WMCO 7.0.1 when it is released. If your WMCO is configured with the *Automatic* update approval strategy, you should change it to *Manual* for WMCO 7.0.0. See the xref:../windows_containers/enabling-windows-container-workloads.adoc#installing-wmco-using-web-console_enabling-windows-container-workloads[installation instructions] for information on changing the approval strategy.
+* WMCO 7.0.0 does not support running in a namespace other than `openshift-windows-machine-operator`. If you are using a custom namespace, it is recommended that you not upgrade to WMCO 7.0.0. Instead, you should upgrade to WMCO 7.0.1 when it is released. If your WMCO is configured with the *Automatic* update approval strategy, you should change it to *Manual* for WMCO 7.0.0. See the xref:../../windows_containers/enabling-windows-container-workloads.adoc#installing-wmco-using-web-console_enabling-windows-container-workloads[installation instructions] for information on changing the approval strategy.
 
+[role="_additional-resources"]
 .Additional resources
 
-See the full list of xref:../windows_containers/windows-containers-release-notes-7-x.html#windows-containers-release-notes-limitations_windows-containers-release-notes[known limitations]
+See the full list of xref:../../windows_containers/wmco_rn/windows-containers-release-notes-7-x-past.adoc#windows-containers-release-notes-limitations_windows-containers-release-notes-7-x-past[known limitations]
 
 include::modules/wmco-prerequisites.adoc[leveloffset=+1]
```

Comment:
Just looking at it from a functional perspective, you might consider moving the prerequisites up a level, so they're not under "past releases" - as a user, that might not be where I would look for them. I would expect to see them under the same level as "Getting Support" or even under the new, current release notes page. Just a suggestion - feel free to ignore if the location makes more sense to you there.